app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -193,7 +194,7 @@ abstract class BaseBrowserFragment : Fragment(), BackHandler, SessionManager.Obs
 
             findInPageIntegration.set(
                 feature = FindInPageIntegration(
-                    sessionManager = requireComponents.core.sessionManager,
+                    sessionManager = context.components.core.sessionManager,
```

Comment:
nit: This could just be `sessionManager = sessionManager` as we have val declared on line 152?